tests/upstream-e2e/test-driver.yaml

```diff
@@ -2,6 +2,7 @@ StorageClass:
   FromName: true
 DriverInfo:
   Name: linodebs.csi.linode.com
+  SupportedFsType: {"ext4", "xfs"}
```

Comment:
Are there any other fsTypes we need to include? 

I see that azure has a quite a few. Not sure if we support some them as well.

Azure link: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/a8ed69636132b9cf922fd07ac06a64d8e71780a5/test/external-e2e/manifest/testdriver.yaml#L7

Reply:
Updated to include ext2 and ext3 as well.